src/Components/ClusterHeader/ClusterHeader.spec.ct.js

```diff
@@ -13,6 +13,7 @@ const LAST_SEEN_FIELD = '#cluster-header-last-seen > :nth-child(2)';
 describe('cluster page header', () => {
   let props;
 
+  //FIXME where is clusterId computed?
```

Comment:
what do you mean? we receive cluster ids in the API response

Reply:
I think no endpoint returns "clusterID" as field. Probably it is copied from another field in the API response and then converted in the format used by the object. Where is that conversion done?

Reply:
Oh, gotcha. The `clusterId` property is sent from src/Components/ClusterHeader/index.js, where it is directly taken from URL path part (`match.params.clusterId`). In other words, once users are at advisor/clusters/foobar page, then the `clusterId` has the value `foobar`. And this has nothing to do with API 👍🏼

Reply:
OK, understood. Thank you!